pkgs/tools/networking/curl/default.nix

```diff
@@ -39,6 +40,8 @@ assert gnutlsSupport -> gnutls != null;
 assert gsaslSupport -> gsasl != null;
 assert gssSupport -> libkrb5 != null;
 assert http2Support -> nghttp2 != null;
+assert http3Support -> nghttp3 != null;
+assert http3Support -> ngtcp2 != null;
```

Comment:
Those two lines do nothing because the inputs do not have ``? null``. We can always override curl with ``http3Support = false`` to not draw the libraries in which is preferred because it has less bug hiding potential.